internal/helper/maps.go

```diff
@@ -0,0 +1,24 @@
+package helper
+
+// Maps recursively merges multiple maps of type map[string]any.
+func Maps(p map[string]any, s ...map[string]any) map[string]any {
```

Comment:
If this is internal why not just map it private? 

```suggestion
func maps(p map[string]any, s ...map[string]any) map[string]any {
```

Feel free to resolve this comment because the function is in a `internal` package, so it's pretty clear it shouldn't be used or depended on, but generally I would just put these functions in the main package and make them private

Reply:
`helper.Maps` is being used on multiple packages, not just one. We generally want to avoid copy-and-pasting code on multiple packages.

<img width="316" alt="image" src="https://github.com/user-attachments/assets/3cb18063-7752-4015-8e72-1f51aa31b930" />

The `internal` directory is a Go idiom: https://dev.to/stefanalfbo/internal-packages-in-go-2h9